pkg/adapter/v2/config.go

```diff
@@ -70,12 +70,12 @@ type EnvConfig struct {
 	// This is used to configure the metrics exporter options,
 	// the config is stored in a config map inside the controllers
 	// namespace and copied here.
-	MetricsConfigJson string `envconfig:"K_METRICS_CONFIG" required:"true"`
+	MetricsConfigJson string `envconfig:"K_METRICS_CONFIG" default:""`
```

Comment:
Can we provide a default value that does not produce an error? For `K_LOGGING_CONFIG` an empty json (`{}`) should work.